pilz_testutils/include/pilz_testutils/async_test.h

```diff
@@ -73,20 +74,24 @@ class AsyncTest
     void triggerClearEvent(const std::string &event);
 
     /**
-     * @brief Will block until the event given by clear_event is triggered. Unblocks immediately, if the event was
-     * triggered in advance.
+     * @brief Will block until the event given by clear_event is triggered or a timeout is reached.
+     * Unblocks immediately, if the event was triggered in advance.
      *
      * @param clear_event Event that allows the test to pass on
+     * @param timeout_ms Timeout [ms]. If negative, no timeout is used.
```

Comment:
I would say, the timeout beeing negative is an internal thing. Coul we not just say, it is an optional parameter. Then, users of the method just not set it, if they want no timeout.
```suggestion
     * @param timeout_ms Timeout [ms]. (optional)
```

Reply:
Thanks. Sounds good.